lib/PuppeteerSharp/Page.cs

```diff
@@ -200,9 +200,9 @@ public int DefaultTimeout
 
         internal bool IsDragging { get; set; }
 
-        internal bool HasPopupEventListeners => Popup?.GetInvocationList().Any() == true;
+        internal bool HasPopupEventListeners => Popup?.GetInvocationList().Length > 0;
```

Comment:
The difference between using ` != 0` and ` > 0`
![image](https://github.com/hardkoded/puppeteer-sharp/assets/919634/011e3bf0-46ef-43e2-a2df-5901643bb217)

[SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABATARgLABQhxRpJ5ZlF1VtwEEANgAQBiDWAFACIyMwBzAIYAXGAG0AusyFQoASmYBeAHwy5AfgB0AGRgA7ASIAWzNQAYA3DRul6TNgwDMPPoNETpshcrXfteoYmzACESsxWQA)

Reply:
Makes sense